contracts/token/psp1155/traits.rs

```diff
@@ -64,6 +64,17 @@ pub trait IPSP1155 {
     );
 }
 
+#[brush::trait_definition]
+pub trait IPSP1155Mint {
+    /// Creates a new token.
```

Comment:
I guess it doesn't really create a new token per say, it just 'mints' it (increases the emission), just like 'burn' below doesn't delete the token, just decreases the emission/supply